packages/flutter_tools/bin/fuchsia_tester.dart

```diff
@@ -145,7 +145,7 @@ Future<void> run(List<String> args) async {
           BuildMode.debug,
           '',
           treeShakeIcons: false,
-          packagesPath: globals.fs.path.normalize(globals.fs.path.absolute(argResults[_kOptionPackages] as String)),
+          packageConfigPath: globals.fs.path.normalize(globals.fs.path.absolute(argResults[_kOptionPackages] as String)),
         ),
```

Comment:
Since we are already going to be creating a diff on this line, I think we should take this opportunity to format it.
```suggestion
          packageConfigPath: globals.fs.path.normalize(
            globals.fs.path.absolute(argResults[_kOptionPackages] as String),
          ),
```

Reply:
Done